nc-news-fe/src/components/CommentSection.jsx

```diff
@@ -12,14 +16,50 @@ function CommentSection({ article_id, comment_count }) {
       )
       .then((response) => setComments(response.data.articleComments))
       .catch((error) => console.log(error));
-  }, [article_id]);
+  }, [newCommentCount]);
+
+  const handleNewComment = (e) => {
+    e.preventDefault()
+    setIsPosting(true)
+    axios.post(`https://be-nc-news-sopv.onrender.com/api/articles/${article_id}/comments`, {
+      username: "grumpy19",
+      body: newComment,
+    })
+    .then((response) => {
+      setNewComment("")
```

Comment:
We could get caught in our error state. If you set error to be null here, this would be avoided